### ISSUE TITLE
check internet connection

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature-packaging#2]
Peut-être faudrait-il vérifier le fonctionnement de la connection internet durant l'install de geonature-db (qui fonctionne bien par ailleurs, beau boulot!). car l'installation nécessite le chargement des données de référentiels, a moins qu'il ne soit prévu de les intégrer dans le paquet.

**Comment by gildeluermoz** [comment]
Merci Fred. Oui, bonne idée. Demain je regarde ça. 
J'ai fini la partie "bourin" de transcription du install_db.sh. Reste qq bug et toute la logique d'update + qq tests permettant de lever des lièvres de ce genre.

**Comment by gildeluermoz** [comment]
En fait, est-ce que tu as déjà installer un paquet sans avoir accès à internet ?
Je sais que c'est possible mais un serveur geonature qui n'aurait pas d'accès à internet, c'est pas très utile !

**Comment by lpofredc** [comment]
Oui, c'est marginal, mais ça a pu arriver. Concrètement, ce n'est pas nécessairement prioritaire car ne concernera que d'hypothétiques cas marginaux. Mais oui,  je connais des personnes/structures qui aujourd'hui encore veulent tout garder sous clé, hors réseau ;)

Sur le principe, a ma connaissance, je ne connais pas de packages `.deb` qui nécessitent le téléchargements de données au cours de l'install, tout est inclu dans les archives. Peut-être envisager un ou plusieurs paquets spéficiques qui ne contiendraient que les script sql d'insert à charger en bdd ?

Avec un paquet unique:  
* `geonature_db_datas` (avec différentes versions de données (i.d. versions de TaxRef, RefGeo ?)

voire mieux avec différents paquets spéficiques permettant d'adapter vraiment l'install à son besoin :
* `geonature-db-datas-taxref13`
* `geonature-db-datas-refgeo2019`
